{{cookiecutter.repo_name}}/.github/workflows/gh-ci.yaml

```diff
@@ -0,0 +1,258 @@
+name: GH Actions CI
+on:
+  push:
```

Comment:
we probably should encourage Cron jobs tbh, the fact that we don't in core is a bug rather than a feature

Reply:
I'd like to either make this an option or get more opinions -- back when I used the MolSSI cookiecutter a lot, I would use up my "GitHub Actions" allotment really fast from pointless cron jobs that I had to remember to turn off. At least IMO should probably be maximum weekly, maybe fortnightly? Either the project is getting updated enough that tests are already running, or it's not getting updated enough that testing at larger intervals is probably fine.

Reply:
O yeah definitely weekly - nightly cron jobs are overkill imho.

Reply:
Added weekly cron jobs